tests/phpunit/tests/st/test-wpml-pb-string-registration.php

```diff
@@ -17,6 +17,7 @@ public function register_shortcode_string( $string_name ) {
 		$post->ID        = $post_id;
 		$post->post_type = 'page';
 		$location        = mt_rand( 1, 100 );
+		$wrap            = '';
```

Comment:
Set a value which is not the default one: `'h1'` maybe.

Reply:
@strategio done